frontend/components/molecules/Activity/index.tsx

```diff
@@ -1,27 +1,57 @@
+import { parseHTML } from '@/helpers/htmlParsing'
+import Link from 'next/link'
+
 type ActivityProps = {
+    usage: 'Question' | 'Answer'
     id: number
+    slug: string
     votes: number
-    is_answered: boolean
-    data: string
+    is_answered?: boolean
+    is_correct?: boolean
+    title?: string
+    content?: string
     created_at: string
 }
 
-const Activity = ({ id, votes, is_answered, data, created_at }: ActivityProps): JSX.Element => {
+const Activity = ({
+    usage,
+    id,
+    slug,
+    votes,
+    is_answered,
+    is_correct,
+    title,
+    content,
+    created_at,
+}: ActivityProps): JSX.Element => {
+    const redirectTo: { Question: string; Answer: string } = {
+        Question: `/questions/${slug}`,
+        Answer: `/questions/${slug}#answer-${id}`,
+    }
+
     return (
-        <div className="flex">
-            <div className="box-border w-14 p-2">
-                {is_answered ? (
-                    <div className="box-border h-full w-full border-2 border-green-600 bg-green-600 text-center text-white">
-                        {votes}
-                    </div>
-                ) : (
-                    <div className="box-border h-full w-full border-2 border-black text-center">
-                        {votes}
-                    </div>
-                )}
+        <div className="flex h-11 flex-row justify-between">
+            <div className="flex w-full flex-row">
+                <div className="box-border min-w-[56px] p-2">
+                    {is_answered || is_correct ? (
+                        <div className="box-border h-full w-full border-2 border-green-600 bg-green-600 text-center text-white">
+                            {votes}
+                        </div>
+                    ) : (
+                        <div className="box-border h-full w-full border-2 border-black text-center">
+                            {votes}
+                        </div>
+                    )}
```

Comment:
How about:

```
<div className={`box-border h-full w-full border-2 text-center ${is_answered || is_correct ? 'border-green-600 bg-green-600 text-white' : 'border-black'}`}>
    {votes}
</div>
```

Reply:
> How about:
> 
> ```
> <div className={`box-border h-full w-full border-2 text-center ${is_answered || is_correct ? 'border-green-600 bg-green-600 text-white' : 'border-black'}`}>
>     {votes}
> </div>
> ```

This is the initial set-up in markup we'll update this as this is much better way to apply the logic